frontend/components/SettingsPage/DebugInfoSection.tsx

```diff
@@ -148,7 +151,7 @@ const DebugItem = ({
 
 export const DebugInfoSection = () => {
   const { masterEoa, masterSafes } = useMasterWalletContext();
-  const { serviceAddresses } = useServices();
+  const { serviceWallets: serviceAddresses } = useServices();
```

Comment:
why change this? serviceAddresses sounds like a list of addresses, not wallets

Reply:
They are wallets, had refactored from higher up, will address